src/App.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import View from './components/View'
```

Comment:
To simplify imports we can create index.ts file inside ./components folder that will contains re-exports from the folder:
```ts
export * from './View';
// ... rest exports from different folders...
```